Dockerfile

```
@@ -78,11 +78,14 @@ RUN mkdir -p /etc/sysconfig/slurm \
         /var/lib/slurmd/qos_usage \
         /var/lib/slurmd/fed_mgr_state 
 
-ARG PUBLIC_KEY
+RUN ssh-keygen -q -t rsa -N '' -f /container_root_pubkey
```

this is a super suggestion. But I would like to keep root organized :DD 

So, I would move to `/root/.ssh`, as normally we users have all of our keys there.

```suggestion
RUN ssh-keygen -q -t rsa -N '' -f /root/.ssh/container_root_pubkey
```